README.md

```diff
@@ -16,7 +16,7 @@
 
 <hr>
 
-[![CI](https://github.com/Belco90/octoclairvoyant/actions/workflows/ci.yml/badge.svg)](https://github.com/Belco90/octoclairvoyant/actions/workflows/ci.yml)
+[![CI](https://github.com/octoclairvoyant/octoclairvoyant-webapp/actions/workflows/ci.yml/badge.svg)](https://github.com/Belco90/octoclairvoyant/actions/workflows/ci.yml)
```

Comment:
```suggestion
[![CI](https://github.com/octoclairvoyant/octoclairvoyant-webapp/actions/workflows/ci.yml/badge.svg)](https://github.com/octoclairvoyant/octoclairvoyant-webapp/actions/workflows/ci.yml)
```

The second link needs to be updated too. 😉

Reply:
Oops! Solved now.